drafts/str4d-overwinter-activation/draft1.rst

```diff
@@ -0,0 +1,196 @@
+::
+
+  ZIP: ???
+  Title: Network Upgrade Mechanism
+  Author: Jack Grigg <jack@z.cash>
+  Comments-Summary: No comments yet.
+  Category: Process
+  Created: 2018-01-08
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "MUST NOT", "SHOULD", and "MAY" in this document are to be interpreted as described in
+RFC 2119.
+
+
+Abstract
+========
+
+This proposal defines a mechanism for coordinating upgrades of the Zcash network, in order to remove ambiguity
+about when network upgrades will activate, provide defined periods in which users should upgrade their local
+software, and minimize the risks to both the upgrading network and those users opting out of the changes.
+
+
+Motivation
+==========
+
+Zcash is a *consensual currency*: nobody is ever going to force someone to use a specific version or a
+specific branch of Zcash. [#consensual-currency]_ As such, different sub-communities will always have the
+freedom to choose different variants or branches which offer different design trade-offs.
```

Comment:
The term "branch" is used. We should define "branch" and "fork" even if the terms can be used interchangeably to avoid any confusion.